src/edu/csus/ecs/pc2/clics/API202306/EventFeedLog.java

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 1989-2025 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.UnsupportedEncodingException;
+
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.log.Log;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+
+/**
+ * Event fee log.
```

Comment:
I didn't know there was a fee for the events... :)

(And yes, I know this is legacy code... ;)

Reply:
I'm wondering if Doug **was** collecting a fee on the side for anyone who used the EF? :-)